### PR TITLE
Starting Landing Page

### PR DESCRIPTION
**Pull request by JerryKu**
Created base components/routes and css for Landing page.

### CODE REVIEW

**Comment by dtarellano** [comment]
Make sure when you do a PR that you have a title of what you did
and in the comment leave some bullets of what you did and maybe screenshot if its something front end wise or if its a bug/error your getting

**Comment by JerryKu** [comment]
Whoops, thought it was just me and Andrew here haha. Yeah I'll keep this in mind :) I'm actually done with the layout for the landing page. Just need to merge this so I can push my commits.

File: client/src/App.jsx [hunk 1]
```diff
@@ -27,12 +28,13 @@ class App extends React.Component {
...
       })
     }
-    
+
     render () {
       return <div className="application">
           <NavbarContainer />
           <Switch>
             <Route exact path="/" component={ActivityFinderContainer} />
+            <Route path="/home" component={LandingPageContainer} />
```
Comment (andrewchen44) [inline client/src/App.jsx]: let's make the landing page container the default path (aka exact path='/') and then change the activity finder container have a path of "/finder"

File: client/src/components/navigating/NavBar.jsx [hunk 2]
```diff
@@ -13,7 +13,8 @@ const HeaderBar = ({ dispatch} ) => {
       <div onClick={createActivity} className='header-tab'> <span>Create an activity</span> </div>
       <NavLink activeClassName="selected" exact to='/about' className='header-tab'> <span>About</span></NavLink>
       <NavLink activeClassName="selected" exact to='/login' className='header-tab'> <span>Login</span></NavLink>
-      <NavLink activeClassName="selected" exact to='/signup' className='header-tab'> <span>Sign Up</span></NavLink> 
+      <NavLink activeClassName="selected" exact to='/signup' className='header-tab'> <span>Sign Up</span></NavLink>
+      <NavLink activeClassName="selected" exact to='/home' className='header-tab'> <span>Home</span></NavLink>
```
Comment (andrewchen44) [inline client/src/components/navigating/NavBar.jsx]: Please change the path of the home page to "/" so it is the default page and add a link to path "/finder" which will be the finding page

File: client/src/containers/LandingPageContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import LandingPage from '../components/landingPage/LandingPage.jsx'
+
+class LandingPageContainer extends Component {
+
+  render() {
+    return (
+      <LandingPage />
+    );
+  }
+
```
Comment (andrewchen44) [inline client/src/containers/LandingPageContainer.jsx]: remove empty space pls

File: client/src/containers/LandingPageContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import LandingPage from '../components/landingPage/LandingPage.jsx'
+
+class LandingPageContainer extends Component {
+
```
Comment (andrewchen44) [inline client/src/containers/LandingPageContainer.jsx]: remove empty line pls

**Pull request closed without merging**